app/controllers/users.rb

```diff
@@ -0,0 +1,37 @@
+get '/sessions/new' do
```

Comment:
these `/sessions/` routes should probably be in a sessions controller file. 

Also matching the route and the template names is a nice conversion. So `erb :"sessions/new"`